translate/rus.json

```diff
@@ -207,6 +207,7 @@
 "last":"последний",
 "left 2":"лево 2",
 "left":"лево",
+"left~2":"осталось",
```

Comment:
This new translation should be used in 'time' section referring to 'time left on the clock'.